src/pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -254,12 +254,13 @@ def on_train_start(self, *_: Any) -> None:
     def on_train_epoch_start(self, trainer: "pl.Trainer", *_: Any) -> None:
         total_batches = self.total_batches_current_epoch
         self.main_progress_bar.reset(convert_inf(total_batches))
+        self.main_progress_bar.initial = 0
```

Comment:
maybe we should reinitialize it to`initial=reloaded value` which is `self.train_batch_idx + self._val_processed` for main progress bar.

Reply:
Setting it to that leads to negative values in the progress bar. 
After one epoch, these counters are outdated.

Reply:
I don't think so they will be outdated since they are reset.

Reply:
Print the values and see for yourself. Doesn't get reset. Hence I can't use it as "initial value". 

<img width="889" alt="image" src="https://user-images.githubusercontent.com/5495193/182344889-670df504-8909-43fe-a4cb-c6ae2480b206.png">

Reply:
ok.. I thought its batch_start hook since it's the case with others.. and there it's 0 after each epoch.